.changeset/wild-aliens-rush.md

```
@@ -0,0 +1,5 @@
+---
+"@siemens/ix-aggrid": patch
+---
+
+Refactor checkbox image mask from Base64 to human readable SVG format.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)